lamstare/utils/single_dptest.sh

```diff
@@ -20,8 +20,8 @@ else
   echo "Model Freezing Finished."
 fi
 
-mv ${frozen_model} ${outdir}
-cd $outdir
+mv ${frozen_model} ${temp_file_path}
+cd $temp_file_path
 
 test_result=${run_id}#${ckpt_name}#${head_name}.txt
```

Comment:
do we need `"` for this, seems it's being interpreted as comments.

Reply:
> do we need `"` for this, seems it's being interpreted as comments.

It's not required, since `#` following a whitespace will be interpreted as comment. But how about using other separators, like `_`?